customize_panther/compose.yaml

```diff
@@ -0,0 +1,33 @@
+x-common-config:
+  &common-config
+  network_mode: host
+  ipc: host
+
+x-cpu-display: &cpu-display
+  env_file: .cpu_display.env
+
+x-gpu-display: &gpu-display
+  runtime: nvidia
+  env_file: .gpu_display.env
+
+
+services:
+  panther_gazebo:
+    image: husarion/panther-gazebo:humble-2.1.2-20241125
+    volumes:
+      - ./config/components.yaml:/config/components.yaml
+    command: >
+      ros2 launch panther_description load_urdf.launch.py
+        components_config_path:=/config/components.yaml
+        add_wheel_joints:=True
+  rviz2:
+    image: husarion/panther-gazebo:humble-ros2-devel-tag-test
```

Comment:
We can use the same image and change command thanks to that we pull only one image, and make sure that all available models in husarion_ugv are also available for in rviz.
```bash
ros2 launch panther_description rviz.launch.py
```

Reply:
actually on this image there was no meshes